action_subscriber.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   end
   spec.add_dependency 'lifeguard'
   spec.add_dependency 'middleware'
+  spec.add_dependency 'thor'
```

Comment:
I was testing the commandline outside of a rails app to make sure the commandline arg passes through correctly to the configuration object and realized that we don't have a declared dependency on thor.

We are clearly depending on it though so I added this explicit dependency